build/js/intlTelInput.js

```diff
@@ -636,11 +647,25 @@ https://github.com/Bluefieldscom/intl-tel-input.git
         },
         // decide where to position dropdown (depends on position within viewport, and scroll)
         _setDropdownPosition: function() {
-            var inputTop = this.telInput.offset().top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
+            var pos = this.telInput.offset(), inputTop = pos.top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
             dropdownFitsBelow = inputTop + this.telInput.outerHeight() + this.dropdownHeight < windowTop + $(window).height(), dropdownFitsAbove = inputTop - this.dropdownHeight > windowTop;
             // dropdownHeight - 1 for border
             var cssTop = !dropdownFitsBelow && dropdownFitsAbove ? "-" + (this.dropdownHeight - 1) + "px" : "";
             this.countryList.css("top", cssTop);
+            // if container calculate postion and append this.dropdown to container
+            if (this.options.dropdownContainer && !this.isMobile) {
+                var inputHeight = !dropdownFitsBelow && dropdownFitsAbove ? 0 : this.telInput.innerHeight();
+                // calculate placement
+                this.dropdown.css({
+                    top: pos.top + inputHeight,
+                    left: pos.left
+                });
+                // close menu on body scroll
+                $("body").off("scroll" + this.ns).on("scroll" + this.ns, function() {
```

Comment:
Shouldn't the $("body").off("scroll") go in _closeDropdown instead?

And then whenever the on scroll event listener fires, you know the dropdown is open, so you don't need to check if countryList.hasClass("hide").